### DISCUSSION TITLE
Review dialog is behind notifications banner on mobile

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#18599] [Pull Requests]
### Steps

1. Have a notification requesting a review
2. Open notifications on mobile
3. Open the notification
4. Tap the add review item
5. Tap the review widget:
    ![Screenshot_20220614-124621](https://user-images.githubusercontent.com/2119212/173632044-57c82532-b33e-473f-b03d-79e470ea75bf.png)

### Actual results
![Screenshot_20220614-124153](https://user-images.githubusercontent.com/2119212/173631552-022120ae-3c55-4381-a5cc-93c6312899aa.png)

**Comment by MathiasReker** [comment]
Please report it here : https://support.github.com/contact/bug-report
